docs/source/legacy.rst

```diff
@@ -64,6 +64,29 @@ To version 0.10
 
 .. note:: StateVector will interact badly with any :code:`numpy` scalar type
 
+Shots in algorithms
+^^^^^^^^^^^^^^^^^^^
+
+When instanciating an algorithm class (:code:`Sampler`, :code:`Analyzer`) with a :code:`RemoteProcessor`, the user now has to
+pass a positive integer value for the named parameter :code:`max_shots_per_call`. Please note that this parameter
+name *has to* be typed in order to avoid potential signature errors.
+
+>>> p = RemoteProcessor("sim:platform")
+>>> sampler = Sampler(p, max_shots_per_call=10_000_000)
+
+This parameter is also handled by local simulations.
+
+.. note:: Probability amplitude back-ends used for sampling (e.g. using **SLOS** for a :code:`sample_count` call) cannot
+    estimate accurately the sample to shots ratio when converting probabilities to samples.
+
+:code:`samples` and :code:`sample_count` parameter was renamed from :code:`count` to :code:`max_samples`
```

Comment:
I don't really understand what has been renamed in what ^^'

Reply:
The code example below doesn't explain it well enough?

Reply:
Do you mean sample_count is also a method ? If so consider putting the sentence "Parameter :code:`count` was renamed :code:`max_samples` in methods :code:`samples` and :code:`sample_count`"